front/pages/api/data_sources/[user]/[name]/documents/index.ts

```diff
@@ -3,28 +3,13 @@ import { authOptions } from "@app/pages/api/auth/[...nextauth]";
 import { NextApiRequest, NextApiResponse } from "next";
 import { unstable_getServerSession } from "next-auth/next";
 import { Op } from "sequelize";
+import {ApiDocument} from "@app/lib/api_models";
 
 const { DUST_API } = process.env;
 
-type Document = {
```

Comment:
not a great fan of using the name `APIDocument` for an internal endpoint. Are the two different? If not I think it should be a `Document` type that we use everywhere?

Reply:
ok I understand your concerns but let me add some context to help the conversation:

I see those "API models" as a way to transport our common api outputs and have the TS type checker help us detect programming errors.
I don't think we should use Sequelize models to do that. Happy to chat more about it but manipulating un-attached ORM object models is a slippery slope to hell.

Maybe the naming here is the main problem?